src/components/feedback/feedback.js

```diff
@@ -147,9 +147,9 @@ class Feedback extends React.PureComponent {
                 </Tooltip>
               </div>
             </div>
-            <div className="flex-child mb6 prose">{children}</div>
+            <div className="mb6 prose">{children}</div>
             {showContactSupport && (
-              <div className="flex-child color-gray">
+              <div className="color-gray">
```

Comment:
Unfortunately color-gray [will not be high enough contrast](https://webaim.org/resources/contrastchecker/?fcolor=617889&bcolor=F6F8F8). I think we removed https://github.com/mapbox/dr-ui/blob/main/src/css/docs-prose.css#L42-L45 during our docs-prose.css sweep. One idea is to use:
```suggestion
              <div className="color-text">
```

It provides a slightly offset gray, but still has a high contrast! Then we won't have to add back CSS.